### ISSUE TITLE
[Bug]: "authCallback id token expired" after upgrade to 14.1.5

### ISSUE THREAD

**Issue by ccondrup** [damienbod/angular-auth-oidc-client#1546]
**What Version of the library are you using?**
14.1.5 on Angular 14.1.3

**Describe the bug**
After upgrading from 14.1.4 to 14.1.5 we're experiencing an issue where users are no longer able to authenticate using Code flow PKCE with refresh tokens. Console says "authCallback id token expired". We are using an inhouse IDP and its API returns an id_token containing these timestamps:

    aut time: Thu Sep 22 2022 10:05:30 UTC+0200 (Central European Summer Time)
    iat time: Thu Sep 22 2022 10:05:34 UTC+0200 (Central European Summer Time)
    exp time: Thu Sep 22 2022 10:10:34 UTC+0200 (Central European Summer Time)

Adding the config param `disableIdTokenValidation: true` enables users to login, but I would like to get to the bottom of this and find the culprit as the timestamps look OK? Please let me know if I can assist with more information or testing.

Config:
```javascript
{
authority: EnvironmentHelper.fetchAPIBase('idp'),
clientId: redacted,
enableIdTokenExpiredValidationInRenew: false,
ignoreNonceAfterRefresh: true,
logLevel: LogLevel.Debug,
postLoginRoute: '/login',
postLogoutRedirectUri: environment.frontend.postLogoutRedirectUri,
redirectUrl: environment.frontend.redirectUrl,
renewTimeBeforeTokenExpiresInSeconds: 666,
responseType: 'code',
scope: 'openid offline_access',
silentRenew: true,
tokenRefreshInSeconds: 300,
triggerAuthorizationResultEvent: true,
useRefreshToken: true,
}
```

**To Reproduce**

1. With 14.1.5, setup Code flow PKCE with refresh tokens, use above config
2. Try to authenticate, witness console error on callback

**Expected behavior**
As the timestamps looks OK to me, I would expect user to get an authorized session like they did on v14.1.4

**Comment by damienbod** [comment]
fixed in version 14.2.0

Once released you can configuration it something like this:

```typescript
import { NgModule } from '@angular/core';
import { AuthModule, LogLevel } from 'angular-auth-oidc-client';

@NgModule({
  imports: [
    AuthModule.forRoot({
      config: {
        authority: '--idp--',
        redirectUrl: window.location.origin,
        postLogoutRedirectUri: window.location.origin,
        clientId: '--client_id--',
        scope: 'openid profile offline_access',
        responseType: 'code',
        silentRenew: true,
        useRefreshToken: true,
        ignoreNonceAfterRefresh: true, // this is required if the id_token is not returned
        allowUnsafeReuseRefreshToken: true, // this is required if the refresh token is not rotated
        enableIdTokenExpiredValidationInRenew: false, // required to refresh the browser if id_token is not updated after the first authentication
        autoUserInfo: false, // if the user endpoint is not supported
        logLevel: LogLevel.Debug,
      },
    }),
  ],
  exports: [AuthModule],
})
export class AuthConfigModule {}
```

**Comment by damienbod** [comment]
fixed in v15.0.0
